code-challenges.test.js

```diff
@@ -38,21 +79,68 @@ const num3 = -7
 
 // b) Create the function that makes the test pass.
 
+//Function will take in a number as an argument and decide if the number is evenly divisible by 3, or not. 
+//Function name: divisibleBy3
+// Input: number
+const divisibleBy3 = (number) => {
+
+    // Use conditional statements with modulo to calculate division, followed by return line with expected output
+    if(number % 3 === 0){
+         return `${number} is divisible by three`
+  } else{
+         return `${number} is not divisible by three`
+  }
+}
+   console.log(divisibleBy3(num1))
+   console.log(divisibleBy3(num2))
+   console.log(divisibleBy3(num3))
 
 
 // --------------------2) Create a function that takes in an array of words and returns an array with all the words capitalized.
 
 // a) Create a test with expect statements for each of the variables provided.
 
+//Test: 
+//Function name: capitalizedArray
+describe("capitalizedArray",() => {
+    //Description of function
+    it("returns an array with all words capitalized",() => {
+      //Function: capitalizedArray, Variable name: randomNouns1, expected output
+      expect(capitalizedArray(randomNouns1)).toEqual["Streetlamp", "Potato", "Teeth", "Conclusion", "Nephew"]
+      //Function: capitalizedArray, Variable name: randomNouns1, expected output
+      expect(capitalizedArray(randomNouns2)).toEqual["Temperature", "Database", "Chopsticks", "Mango", "Deduction"]
+      })
+    
+  })
+// //Yarn jest fail received: 
+// //capitalizedArray › returns an array with all words capitalized
+// //ReferenceError: randomNouns1 is not defined
+
 const randomNouns1 = ["streetlamp", "potato", "teeth", "conclusion", "nephew"]
 // Expected output: ["Streetlamp", "Potato", "Teeth", "Conclusion", "Nephew"]
 const randomNouns2 = ["temperature", "database", "chopsticks", "mango", "deduction"]
 // Expected output: ["Temperature", "Database", "Chopsticks", "Mango", "Deduction"]
 
 
-// b) Create the function that makes the test pass.
+// // b) Create the function that makes the test pass.
 
+// // Return statements 
 
+//Function will take in an array of words and return an array with all the words capitalized.  
+//Function name: capitalizedArray
+// Input: array 
+const capitalizedArray = (array) => {
+    // // Use .map to return an array with the same number of values
+    //use .charAt to locate character at index 0 
+    //use .toUpperCase to capitalize character at index zero on each word 
+    //use .slice to create subset of each word, with letters remaining lowercase 
+    //use .join to separate words by commas 
+    //use .split to split the words by quotation marks
+   let capital = array.map(element => element.charAt(0).toUpperCase() + element.slice(1)).join(" ")
+   return capital.split(" ")
+}
```

Comment:
Logic looks great!  You won't need join and split, since map is already returning the array exactly how you need it.